### PR TITLE
BNH-203, 190 and 196 Property form frontend changes

### PR DESCRIPTION
**Pull request by MaxTheComputerer**
- Rent now appears with property details so the edit button makes sense
- Added button to edit address
- Added radio for single/multiple units and input for number of occupied units
- Added instructions to description
- Added asterisks

image
image
image
image
image
image
image
image

### CODE REVIEW

**Comment by emilypatsko** [comment]
Could you add some screenshots etc. please? :)

**Comment by MaxTheComputerer** [comment]
> Could you add some screenshots etc. please? :)

Ah sorry I completely forgot - they're added now!

File: web/Controllers/PropertyController.cs [hunk 1]
```diff
@@ -113,7 +113,8 @@ public async Task<ActionResult> ViewProperty(int propertyId)
         var owner = LandlordProfileModel.FromDbModel(_propertyService.GetPropertyOwner(propertyId));
         var fileNames = await _azureStorage.ListFileNames("property", propertyId);
         var imageFiles = GetFilesFromFileNames(fileNames, propertyId);
-        var propertyDetailsModel = new PropertyDetailsViewModel { Property = propertyViewModel, Owner = owner, Images = imageFiles };
+        var propertyDetailsModel = new PropertyDetailsViewModel
+            { Property = propertyViewModel, Owner = owner, Images = imageFiles };
```
Comment (emilypatsko) [inline web/Controllers/PropertyController.cs:117]:
```suggestion
        var propertyDetailsModel = new PropertyDetailsViewModel
        {
            Property = propertyViewModel,
            Owner = owner,
            Images = imageFiles 
        };

File: web/ViewModels/PropertyInput/PropertyInputModelDescription.cs [hunk 2]
```diff
@@ -1,30 +1,30 @@
...
-    }
-
-    public override PropertyViewModel FormToViewModel()
-    {
-        return new PropertyViewModel
-        {
-            Description = Description
-        };
-    }
+﻿using System.ComponentModel.DataAnnotations;
```
Comment (emilypatsko) [inline web/ViewModels/PropertyInput/PropertyInputModelDescription.cs:1]: The diff is making it look like this whole file has changed and I'm not sure why - this happened on one of Claire's PRs earlier and I believe she resolved it with Oskar. May be worth checking with Claire how this was fixed

File: web/ViewModels/PropertyInput/PropertyInputModelDetails.cs [hunk 3]
```diff
@@ -1,36 +1,41 @@
...
-
-    public override PropertyViewModel FormToViewModel()
-    {
-        return new PropertyViewModel
-        {
-            PropertyType = PropertyType,
-            NumOfBedrooms = NumOfBedrooms
-        };
-    }
+﻿using System.ComponentModel.DataAnnotations;
```
Comment (emilypatsko) [inline web/ViewModels/PropertyInput/PropertyInputModelDetails.cs:1]: Seems to have happened for this file too

**Review by emilypatsko — CHANGES_REQUESTED** [review]
I think generally this looks good, but it looks like the diff for some of the files is odd. I think the line endings have changed and it's making it look like much more of the file has changed

**Review by emilypatsko — APPROVED** [review]

**Pull request merged**